commit 257556cdcab3377f6cc121adc4124d0b2216150e (refs/changes/51/1551/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2017-02-22T00:30:05+00:00 (2 years, 8 months ago)
    
    MORAY-398 moray caught connecting to deposed peer
    Reviewed by: David Pacheco <dap@joyent.com>
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/package.json b/package.json
index 5477297..80a3b15 100644
--- a/package.json
+++ b/package.json
@@ -18,8 +18,8 @@
         "moray-filter": "1.0.0",
         "libuuid": "0.1.3",
         "lru-cache": "2.5.0",
-        "node-manatee": "git+https://github.com/joyent/node-manatee.git#152d3e1",
-        "manatee": "git+https://github.com/joyent/manatee.git#8615736",
+        "node-manatee": "git+https://github.com/joyent/node-manatee.git#c2ad9ae",
+        "manatee": "git+https://github.com/joyent/manatee.git#86ae52d",
         "moray": "git+https://github.com/joyent/node-moray.git#fd5781b",
         "microtime": "0.5.1",
         "once": "1.3.0",
